app/models/topic.rb

```diff
@@ -58,7 +58,7 @@ def filtered_topic_thumbnails(extra_sizes: [])
 
   def thumbnail_info(enqueue_if_missing: false, extra_sizes: [])
     return nil unless original = image_upload
-    return nil unless original.filesize < SiteSetting.max_image_size_kb.kilobytes
+    return nil if original.filesize >= SiteSetting.max_image_size_kb.kilobytes
     return nil unless original.read_attribute(:width) && original.read_attribute(:height)
```

Comment:
How did this one get through the filter? 🤔

Reply:
Because it passes this rule: https://docs.rubocop.org/rubocop/cops_style.html#styleunlesslogicaloperators
By default, it doesn’t forbid the use of `unless` with logical operators, only the use of mixed logical operators. But we could decide to not use any logical operators when using `unless`. I think we’re fine with the defaults but wdyt?